specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -613,6 +619,103 @@
+        ],
+        "operationId": "Clusters_GetAzureAsyncOperationStatus",
+        "description": "The the async operation status.",
+        "x-ms-examples": {
+          "Get Async Operation Status of Creating Cluster": {
+            "$ref": "./examples/GetClusterCreatingAsyncOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./operations.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/updateClusterIdentityCertificate": {
+      "post": {
+        "tags": [
+          "Clusters"
+        ],
+        "operationId": "Clusters_UpdateIdentityCertificate",
+        "description": "Updates the cluster identity certificate.",
+        "x-ms-examples": {
+          "Update cluster identity certificate": {
+            "$ref": "./examples/HDI_Clusters_UpdateClusterIdentityCertificate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateClusterIdentityCertificateParameters"
+            },
+            "description": "The cluster configurations."
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./operations.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK response definition."
+          },
+          "202": {
+            "description": "Accepted response definition"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Can you also specify x-ms-long-running-operation-options here?
Please refer doc: http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options

Reply:
> Can you also specify x-ms-long-running-operation-options here?
> Please refer doc: http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options

Hi @RamyasreeChakka Thanks for your comment, I have fixed this.